### DISCUSSION TITLE
2019/02/desain-karangan-bunga-bagian-1

### DISCUSSION THREAD

**Discussion by tegursapa** [madi10/MANTANKODE#19] [General]
# 2019/02/desain-karangan-bunga-bagian-1

desain untuk karangan bunga dengan tema hijau cocok untuk ucapan atau bela sungkawa atau duka cita

https://www.mantankode.net/2019/02/desain-karangan-bunga-bagian-1.html?m=1

**Comment by madi10** [comment]
Ada saran? Tema karangan bunga seperti apa lagi yang perlu saya buat?
